src/FwdState.cc

```diff
@@ -27,7 +28,6 @@
 #include "comm/Loops.h"
 #include "CommCalls.h"
 #include "errorpage.h"
-#include "event.h"
```

Comment:
This `event.h` and corresponding `event.cc` code is all part of the `EventLoop` mechanism. Not part of `AsyncJob` code, even though the two are similar and interact.